bijection-avro/src/main/scala/com/twitter/bijection/avro/AvroCodecs.scala

```diff
@@ -109,6 +109,18 @@ object SpecificAvroCodecs {
   }
 
   /**
+   * Returns Injection capable of serializing and deserializing a compiled avro record using org.apache.avro.io.BinaryEncoder.
+   * Accepts a schema to support generated Scala case classes.
+   * @tparam T compiled Avro record
+   * @return Injection
+   */
+  def toBinaryWithSchema[T <: SpecificRecordBase: ClassTag](schema: Schema): Injection[T, Array[Byte]] = {
+    val writer = new SpecificDatumWriter[T](schema)
+    val reader = new SpecificDatumReader[T](schema)
+    new BinaryAvroCodec[T](writer, reader)
```

Comment:
You are right about `ClassTag`, will remove it.

If the schema is similar (e.g. same record name) it seems to kind of work. When it's completely wrong, it throws `java.lang.ClassCastException: org.apache.avro.generic.GenericData$Record cannot be cast to avro.FiscalRecordScala`. If have added a test case and doc string.